weather_mv/loader_pipeline/netcdf_loader.py

```diff
@@ -202,8 +203,10 @@ def to_row(it: t.Dict) -> t.Dict:
         row_ds = data_ds.loc[it]
 
         # Create a Name-Value map for data columns. Result looks like:
-        # {'d': -2.0187, 'cc': 0.007812, 'z': 50049.8}
-        row: t.Dict = row_ds.to_pandas().apply(to_json_serializable_type).to_dict()
+        # {'d': -2.0187, 'cc': 0.007812, 'z': 50049.8, 'rr': None}
+        temp_row = row_ds.to_pandas().apply(to_json_serializable_type)
+        # Pandas coerces floating type None values back to NaNs, need to do an explicit replace after.
+        row: t.Dict = temp_row.astype(object).where(pd.notnull(temp_row), None).to_dict()
```

Comment:
I don't think the type annotation is necessary, since the RHS ends with `to_dict()`. WDYT?

Reply:
If you are referring to astype(object), that is the only way to make NaNs go to None.
  
Floating types are converted back to NaNs and behavior is inconsistent otherwise depending on pandas version
More details here, https://github.com/pandas-dev/pandas/issues/42423

Reply:
That makes sense to me. 

I'm referring to the `t.Dict` annotation, specifically. Since the line ends in `.to_dict()`, it's clear to me that `row` is a dictionary without the annotation.

Reply:
done